packages/ui/src/icons/icon/icon-with-activity.native.tsx

```diff
@@ -0,0 +1,19 @@
+import { View } from 'react-native';
```

Comment:
Sometimes we need to show an activity type icon on top of a protocol item, and sometimes we need to show a protocol icon on top for example here:
![Screenshot 2024-11-26 at 09 23 58](https://github.com/user-attachments/assets/66723cd9-e0ec-4ebe-a110-cb8e9e43a4fe)

**NOTE** the design is wrong for the first two items - Stacks + Btc. 

For subsequent tokens like `sBTC`, `ALEX Lab` etc. we need to show a base protocol icon on top of their specific item. 

Maybe we can extend this component to be more broad and handle both activity and protocol sub-icons

Reply:
Yeah that sounds good, should be somewhat easy to achieve. 

To that point I notice we are hard coding our protocol icons - I would love to see us use something more generic like a `<ProtocolIcon symbol="btc | stx | ..." />`